packages/headless/src/features/advanced-search-queries/advanced-search-queries-state.ts

```diff
@@ -11,6 +11,8 @@ export interface AdvancedSearchQueriesState {
 }
 
 export const getAdvancedSearchQueriesInitialState: () => AdvancedSearchQueriesState = () => ({
-  cq: '',
+  cq: null,
   aq: '',
 });
+
+export const getConstantQueryDefaultValue = () => '';
```

Comment:
I don't really like this, but my imagination kinda fails me on that one... Should this be a constant or a function? What should it be named? Does it belong somewhere else? Alternatively, it could be removed and replaced with `?? ""` whenever anything checks for null.